src/game/client/components/stats.h

```diff
@@ -29,6 +57,15 @@ class CStats: public CComponent
 	virtual void OnConsoleInit();
 	virtual void OnRender();
 	virtual void OnMessage(int MsgType, void *pRawMsg);
+
+	void UpdatePlayTime(int Ticks);
+	void OnMatchStart();
+	void OnFlagGrab(int ClientID);
+	void OnFlagCapture(int ClientID);
+	void OnPlayerEnter(int ClientID, int Team);
+	void OnPlayerLeave(int ClientID);
+
+	const CPlayerStats *GetPlayerStats(int ClientID) const { return &m_aStats[ClientID]; }
 };
 
 #endif
```

Comment:
Newline would be nice. Also the whole file seems to have bugged line endings that show a ^M in the git diff.